google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -834,6 +834,15 @@ public static BlobListOption currentDirectory() {
     }
 
     /**
+     * Returns an option for bucket's billing user project. This option is required for and only
```

Comment:
This comment doesn't match comment above.

Reply:
Done.